web/src/compose_fade.js

```diff
@@ -18,22 +18,27 @@ export function set_focused_recipient(msg_type) {
 
     // Construct focused_recipient as a mocked up element which has all the
     // fields of a message used by util.same_recipient()
-    const focused_recipient = {
-        type: msg_type,
-    };
-
-    if (focused_recipient.type === "stream") {
+    let focused_recipient;
+    if (msg_type === "stream") {
         const stream_id = compose_state.stream_id();
-        focused_recipient.topic = compose_state.topic();
+        const topic = compose_state.topic();
```

Comment:
I moved these to `const` definitions here, rather than the `let` definitions you'd had above, which were unnecessary and a bit confusing.